src/hist/axis.py

```diff
@@ -30,7 +70,7 @@ def __init__(
         else:
             raise Exception("Name should be a valid Python identifier")
         metadata["title"] = title
-        super(bha.Regular, self).__init__(
```

Comment:
In Python 3, I would always use `super()` by itself. It's easy to make a mistake. Such as here - `super` takes the *current* class, not the parent. In boost-histogram 0.8.0, I had these double-wrapped, so you didn't notice the "one up", expect for the `__repr__` not working. But in 0.9.0, there is no double-wrapping, so this was grabbing the wrong class. You could fix it by using `super(Regular, self)`, but better yet, just use `super()`.